Intrepid.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "Intrepid"
-  s.version       = "0.9.2"
+  s.version       = "0.9.3"
```

Comment:
We might want to go to `0.10.0` since this is a breaking change that will no longer compile in xcode 9.2 <